src/pages/youthboard/villages/index.tsx

```
@@ -9,32 +9,42 @@ import SearchBar from '@/components/Searchbar';
 import SortBy from '@/components/youthNet/SortBy';
 import YouthAndVolunteers from '@/components/youthNet/YouthAndVolunteers';
 import {
+  DROPDOWN_NAME,
   users,
```

ESLint

[error] 'users' is defined but never used.

(no-unused-vars)